### PR TITLE
Enhance docker

### PR DESCRIPTION
**Pull request by babsey**
This PR renames the folder for docker files and use `nginx:stable-alpine` as base image.

### CODE REVIEW

File: docker/Dockerfile-builder [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Name the stage "builder"
+FROM node:16-alpine AS builder
+
+COPY . .
+
```
Comment (jens-br) [inline docker/Dockerfile-builder:5]: What was the reason for this line?
```suggestion
```

**Review by jens-br — COMMENTED** [review]
A question remains, rest seems good.

File: docker/Dockerfile-builder (hunk 1 above)
Comment (babsey) [reply docker/Dockerfile-builder:11]: I added `WORKDIR /app` in DockerFile.

**Review by jens-br — COMMENTED** [review]
(a small question, please see below)

**Pull request merged**